lisa/trace.py

```diff
@@ -310,16 +370,19 @@ def has_events(self, events):
         else:
             return set(events).issubset(set(self.available_events))
 
+    def get_view(self, window):
+        return TraceView(self, window)
+
```

Comment:
Could be interesting to also expose it as:
```
    def __getitem__(self, window):
        if not isinstance(window, slice):
             raise TypeError('window must be a slice')
        if slice.step is not None:
            raise ValueError('step is not handled')
        return TraceView(self, (slice.start, slice.stop))
```
This enables slicing on the `Trace` object:
```
view = Trace(...)[start:stop]
```

note: that also needs to be implemented on `TraceView` (ideally shared in a base class like `TraceHelpers`), since `__getattr__` will not handle lookup of `__getitem__` when it is done by the interpreter (same goes for all the special methods).

Reply:
Hooo yes that would be pretty damn great!

Reply:
Now that I think about it, the non-slice case could be used as an alias to `df_events()`

Reply:
I'm sold on the slice (and I think I'll remove `get_window` from `Trace`), but I'm not sure if we want to depart from `df_events()`. @derkling ?

Reply:
I would keep `df_events` as well, it's would be just some kind of shorthand (exposing things a non-magic that are always looked up as expected is a good idea from my experience).

Reply:
I'll go for just the slice for now, and we can always add the `df_events` alias later on.